src/PhpWord/Style/Border.php

```diff
@@ -22,6 +22,41 @@
  */
 class Border extends AbstractStyle
 {
+    /**
```

Comment:
Please move to a PHPWord/SimpleType/Border
source is http://www.datypic.com/sc/ooxml/t-w_ST_Border.html